dts/bindings/mtd/mspi-atxp032.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024, Ambiq Micro Inc. <www.ambiq.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: NOR Flash devices ATXP032 on MSPI bus
+
+compatible: "mspi-atxp032"
+
+include: ambiq,mspi-device.yaml
```

Comment:
Shouldn't this be `mspi-device.yaml`?

Reply:
Yes, it got left out as an unresolved issue. The ATXP032 flash device driver is intended to be a universal driver, but I needed to pass some properties that is unique to Ambiq from DTS. Now that I think of it, I should be able to add the ambiq,mspi-device compat to the DTS instead of including it here.